kroxylicious-operator/pom.xml

```diff
@@ -322,4 +322,143 @@
             </testResource>
         </testResources>
     </build>
+
+    <profiles>
```

Comment:
It's worth saying that there's no especial reason to have the kube manifests in the `kroxylicious-operator` module. I.e. we could take the approach that `kroxylicious-operator` is everything upto and including the ITs, but do the packaging in a separate module in which we keep the manifests, and do the in-kube testing.

Reply:
I like the idea of a separate module per packaging type - yaml-bundle, olm-bundle etc. However, some of the inputs will be shared (roles, service accounts, etc), some will be particular to the packaging type e.g Deployment for the yaml bundle, CSV for the olm-bundle.  Maybe we need a place to keep the common stuff, which the packaging modules can rely on.